stl/inc/format

```diff
@@ -1352,6 +1393,13 @@ struct _Format_handler {
     }
 };
 
+// Generic formatter definition, the deleted default constructor
+// makes it "disabled" as per [formatter.requirements]/1.1.1

```

Comment:
Nitpick: We should cite Working Papers along with stable names, because paragraphs get renumbered over time.